test/test_backoff.py

```diff
@@ -186,11 +194,35 @@ def test_backoff_randomness(self, random_patch, backoff_with_random_sigma):
                 'count': 1
             }]
         }
-        assert backoff.get_next_expiration(message, "backoff") == 2200
+        backoff.calculate_next_expiration(message, "backoff")
+        assert backoff.next_expiration == 2200
         assert random_patch.gauss.call_args_list == [
             call(2000, backoff.random_sigma)
         ]
 
+    def test_uncalculated_to_string(self):
+        backoff = Backoff()
+
+        assert str(backoff) == (
+            "<class 'nameko_amqp_retry.backoff.Backoff'>(not-yet-calculated)"
```

Comment:
Do we need the full repr here? `nameko_amqp_retry.backoff.Backoff(msg)` or even `Backoff(msg)` may be sufficient